<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundForm</class>
 <widget class="QDialog" name="BackgroundForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="BackgroundMode">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="AutomaticSheet">
      <attribute name="title">
       <string>Automatic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="LinearRegButton">
            <property name="text">
             <string>Linear regression between</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LRFirst"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>and</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LRLast"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AverageButton">
            <property name="text">
             <string>Average over</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AverageNPoints"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>points starting at</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AverageStart"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>83</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="CalcAutoButton">
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteAutoButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ManualSheet">
      <attribute name="title">
       <string>Manual</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="BkgrPointsList">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>19</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>18</number>
         </attribute>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteAllButton">
         <property name="text">
          <string>Delete all</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
